src/encoded/commands/ingest_vcf.py

```diff
@@ -33,6 +33,8 @@ def main():
                         help='Provide if consequences should be uploaded')
     parser.add_argument('--post-genes', action='store_true', default=False,
                         help='Provide if genes should be uploaded')
+    parser.add_argument('--structural_variant', action='store_true', default=False,
+                        help='Provide if ingestion SV VCF')
```

Comment:
Should be hyphenated for args, as that's easier for most people to remember and type, not to mention all the other arguments are hyphenated, so one exception is _especially_ hard to remember. If you've made a bunch of scripts relying on underscores that you're worried about breaking, it's OK to do this to tolerate both (untested):
```suggestion
    parser.add_argument('--structural-variant', '--structural_variant',
                        dest='structural_variant',
                        action='store_true', default=False,
                        help='Provide if ingestion SV VCF')
```